README.md

```
@@ -238,7 +239,8 @@
   
   <br>
   
-  ## 🗂 データベース設計図（物理ERD / Rails前提）
+  
+  ## �🗂 データベース設計図（物理ERD / Rails前提）
```

markdownlint-cli2 (0.20.0)

Headings must start at the beginning of the line

(MD023, heading-start-left)